credentials/credentials.go

```diff
@@ -87,6 +88,7 @@ func (s SecurityLevel) String() string {
 // This API is experimental.
 type CommonAuthInfo struct {
 	SecurityLevel SecurityLevel
+	SPIFFEID      *url.URL
```

Comment:
I wouldn't consider SPIFFE to be "common" to all transport security implementations.  I don't think it makes as much sense here, at least not if the only reason to move it was because this struct is already marked as experimental.

Reply:
Got it. I put it back to TLSInfo. Since TLSInfo itself is not EXPERIMENTAL, I marked only the SPIFFEID EXPERIMENTAL. Please let me know if it works:)